testsuite/tests/typing-local/unregion.ml

```diff
@@ -0,0 +1,74 @@
+(* TEST
+ * expect *)
+
+(* TYPING TESTS *)
+(* TYPING TESTS *)
+(* TYPING TESTS *)
+(* the following two tests show that y is regional *)
```

Comment:
This comment is a little confusing. It isn't really that `y` is regional, it's that it is local for the caller's region rather than this function's region. I'd probably reword this to something like "the following two tests show that y can leave the current function despite being local".